src/utils/hints/microblocksHints.js

```diff
@@ -0,0 +1,8 @@
+export const microblocksHints = {
+  microblock: 'Microblocks can be created every 3 seconds by the current leader, are linked to a specific keyblock and include all types of on-chain transactions. The next leader is incentivized to keep the microblocks in the blockchain as he receives 40% of the fees of the transactions.',
+  keyblock: 'Parent keyblock id in which the microblock is linked.',
+  microblockHeight: 'Parent keyblock height in which the microblock is linked.',
```

Comment:
```suggestion
  microblockHeight: 'Parent keyblock height to which the microblock is linked.',
```

Looks better to me

Reply:
This was marked as resolve but I still see `in`

Reply:
Sorry, my bad. Fixed